cl/cl/src/bundle.rs

```diff
@@ -24,11 +18,11 @@ pub struct BundleWitness {
     pub balance_blinding: Scalar,
 }
 
-#[derive(Debug)]
-pub struct BundleProof {
-    pub partials: Vec<PartialTxProof>,
-    pub balance_blinding: Scalar,
-}
+// #[derive(Debug)]
```

Comment:
Ah, yep meant to remove these. All these proof structs and logic are moving into the ledger crate. This is to avoid a circular dependency where the risc0 proof depends on cl which in turn depends on risc0 for the proofs.